superset/views/core.py

```diff
@@ -857,6 +864,26 @@ def addtodashboard(self):
                 table_response = req_session.post(request.host_url+'tablemodelview/create' ,headers = headers,data = request.form,params=params)
                 _slice['datasource'] =  json.loads(table_response.content)['table_name']
 
+                _slice = update_slice_metadata(_slice)
+
+                slice_param_data = {
+                  'action':	'saveas',
+                  'slice_name':	_slice['slice_name'],
+                  'add_to_dash':	'existing',
+                  'save_to_dashboard_id':	json.loads(new_dashboard.content)['dashboard_id'],
+                  'goto_dash':	'false',
+                }
+
```

Comment:
remove white spaces  in property and value

Reply:
done